test/pkgs/gitea-dev/Kptfile

```diff
@@ -0,0 +1,22 @@
+apiVersion: kpt.dev/v1
+kind: Kptfile
+metadata:
+  name: gitea-dev
+  annotations:
+    config.kubernetes.io/local-config: "true"
+upstream:
+  type: git
+  git:
+    repo: https://github.com/efiacor/test
```

Comment:
Does this package differs from the gitea one in catalog being a slimmer installation?

Reply:
Oops. That shouldn't be there. Good catch! Thanks